gallery/src/components/NxFontAwesomeIcon/NxFontAwesomeIconExample.tsx

```diff
@@ -27,6 +27,12 @@ const NxFontAwesomeIconExample = () =>
         */}
       <NxFontAwesomeIcon icon={faBatteryEmpty} color="red" />
     </button>
+    {/* Icon only variant button with aria-label on the icon, note that aria-hidden has been set to false */}
+    <NxButton variant="icon-only">
+      <NxFontAwesomeIcon icon={faEdit} aria-hidden="false" aria-label="Edit text" />
```

Comment:
Should we automatically set aria-hidden to false in NxFontAwesomeIcon when an aria-label prop is set instead of manually doing it here?

Reply:
if we set aria-hidden automatically, we can revisit lots of other style and component code to remove aria-hidden.

Reply:
> Should we automatically set aria-hidden to false in NxFontAwesomeIcon when an aria-label prop is set instead of manually doing it here?

That's an interesting idea and would streamline things, I'll take a look at that.